applications/nrf5340_audio/src/modules/sd_card.c

```diff
@@ -222,3 +230,72 @@ int sd_card_init(void)
 
 	return 0;
 }
+
+static volatile bool seg_read_started = false;
+static struct fs_file_t f_seg_read_entry;
+
+int sd_card_segment_read_open(char const *const filename, const char *path_to_file)
+{
+	char abs_path_name[PATH_MAX_LEN + 1] = SD_ROOT_PATH;
+	strcat(abs_path_name, path_to_file);
+	printk("abs path name:\t%s\n", abs_path_name);
```

Comment:
printk is used a lot in Zephyr, but we use LOG_DBG, LOG_INF, LOG_WRN and LOG_ERR

Reply:
And this should probably be a LOG_DBG